plugins/plugin-optimize/test/stubs/minimal/index.html

```diff
@@ -1 +1,13 @@
-<!DOCTYPE html><html lang="en"><head><meta charset="UTF-8"><title>Minimal test</title><link rel="stylesheet" href="style.css"></head><body><script type="module" src="index.js"></script></body></html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Minimal test</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <script type="module" src="index.js"></script>
+    <!-- <script type="module" src="./do-not-preload-1.js"></script> -->
+    <script type="module" src="./do-not-preload-2.js"></script>
+  </body>
+</html>
```

Comment:
Expanded the HTML, a) so we can test minification, and b) so the output is clearer when not minifying